addons/disable-warning-not-save/addon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Disable warning not save",

```

Comment:
Maybe we can change name a little bit?
For example `Disable warning about closing not saved project` or something like that.

Reply:
`Disable leave editor warning`?